app/controllers/general.php

```diff
@@ -360,6 +360,11 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
             'APPWRITE_VCS_ROOT_DIRECTORY' => $deployment->getAttribute('providerRootDirectory', ''),
         ]);
 
+        // SPA fallbackFile override
+        if ($resource->getAttribute('adapter', '') === 'static' && $resource->getAttribute('fallbackFile', '') !== '') {
```

Comment:
Instead of having a special env var here, can we just overwrite the path we sent to the runtime? Won't it make the rest of the flow simpler?

Reply:
What we are setting is just a path, inside build output directory of developer.
Static server inside runtime then recieves this env var, so it can do proper SPA routing

Reply:
This is valid only to the SPA runtime, right?